app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -416,8 +417,8 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
         if (region.coverage > 0.0D) {
           val properties: JsObject = Json.obj(
             "coverage" -> region.coverage,
-            "region_id" -> region.regionID,
-            "region_name" -> region.name,
+            "Neighborhood ID" -> region.regionID,
```

Comment:
We should stick with JSON convention here and do `neighborhood_id` and `neighborhood_name`